client/components/app.jsx

```diff
@@ -6,31 +6,62 @@ import CarouselPic from './carousel.jsx';
 import DisplayPic from './display.jsx';
 import LeftArrow from './leftArrow.jsx';
 import RightArrow from './rightArrow.jsx';
+import HeartButton from './heartButton.jsx';
```

Comment:
Please see https://github.com/airbnb/javascript/blob/master/react/README.md#naming re: file names (and also configure ESLint for Airbnb's style guide asap).